show/p4_table.py

```diff
@@ -0,0 +1,34 @@
+import json
+import click
+import utilities_common.cli as clicommon
+from swsscommon.swsscommon import SonicV2Connector
+from tabulate import tabulate
+
+
+#
+# 'p4_table' command ("show p4-table")
+#
+@click.command()
+@click.argument('table_name', required=False)
+@click.option('--verbose', is_flag=True, help='Enable verbose output')
+def p4_table(table_name, verbose):
+  """Display all P4RT tables"""
+  appDB = SonicV2Connector(host='127.0.0.1')
+
+  if appDB is None:
+    click.echo('Failed to connect to the application database.')
+    return -1
+
+  db = appDB.APPL_DB
+  appDB.connect(db)
+
+  if table_name is None:
+    table_name = ''
+
+  keys = appDB.keys(db, f'P4RT_TABLE:{table_name}*')
```

Comment:
>P4RT_TABLE: [](http://example.com/codeflow?start=26&length=11)

If you use `Table` class, you do not need to concatenate table name and entry name.

Reply:
`:` as separator is defined in database_config.json

Reply:
Are you referring to Table class in swsscommon? This class takes swsscommon.DBConnector as an argument rather than swsscommon.SonicV2Connector, so it appears that I cannot use the provided mock dbconnector in my unit test if I use this class.

 Will it suffice to match on the separator defined in database_config.json?